crates/ruff_python_formatter/src/snapshots/ruff_python_formatter__tests__black_test__comments2_py.snap

```diff
@@ -356,10 +356,8 @@ instruction()#comment with bad spacing
 +    lcomp3 = [i for i in []]
      while True:
          if False:
--            continue
-+            NOT_YET_IMPLEMENTED_StmtContinue
- 
-             # and round and round we go
+             continue
+@@ -141,25 +104,13 @@
```

Comment:
The ruff and black output both spit out:

```py
while True:
    if False:
        continue

        # and round and round we go
    # and round and round we go
```

cc: https://discord.com/channels/1039017663004942429/1073384953741574217/1119620838161928342

I wanna wrap my head around this, but I originally read this as "the diff excludes the comment"

Reply:
Makes sense. "This is no longer in the diff" not "the output excludes this".

Reply:
yeah the diff of diffs is confusing unfortunately